app/views/users/change_username.html.erb

```diff
@@ -1,12 +1,23 @@
 <!--Descriptive page name, messages and instructions-->
 <h2 class="heading"><%= ts("Change My User Name") %></h2>
 <%= error_messages_for :user %>
-<p class="caution">
-  <%= ts("Please use this feature with caution. For information on how changing your user name will affect your account, please check out the %{account_faq}. Note that changes to your username may take several days or longer to appear. If you are still seeing your old user name on your works, bookmarks, series, or collections after a week, please %{contact_support}.",
-  account_faq: (link_to ts("Account FAQ"), archive_faqs_path + "/your-account#namechange"),
-  contact_support: (link_to ts("contact Support"), new_feedback_report_path)
-  ).html_safe %>
-</p>
+<div class="caution">
+  <p>
+    <strong><%= t("users.change_username.caution") %></strong>
+    <%= t("users.change_username.change_window",
+      times: t("users.change_username.times", count: ArchiveConfig.USER_RENAME_LIMIT),
+      days: distance_of_time_in_words_to_now(ArchiveConfig.USER_RENAME_LIMIT_DAYS.days.ago)) %>
+    <% if @user.renamed_at %>
+      <%= t("users.change_username.last_renamed", renamed_at: l(@user.renamed_at, format: :long)) %>
+    <% end %>
+  </p>
+  <p>
+    <%= t("users.change_username.more_info",
+    account_faq_link: (link_to t("users.change_username.account_faq"), archive_faq_path("your-account", anchor: "namechange")),
```

Comment:
Can you indent these lines similar to the `users.change_username.change_window` section?